src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -110,37 +112,37 @@
 
     }
 
-    @ApiOperation(value = "Retrieves user profile with roles by id",
-                  authorizations = {
+    @ApiOperation(value = "Retrieves a User Profile and their Roles with the given ID",
+            authorizations = {
                     @Authorization(value = "ServiceAuthorization"),
                     @Authorization(value = "Authorization")
-                  }
+            }
     )
     @ApiResponses({
-        @ApiResponse(
-            code = 200,
-            message = "Representation of a user profile data",
-            response = String.class
-        ),
-        @ApiResponse(
-            code = 400,
-            message = "Bad Request",
-            response = String.class
-        ),
-        @ApiResponse(
-            code = 404,
-            message = "Not Found",
-            response = String.class
-        ),
-        @ApiResponse(
-            code = 500,
-            message = "Internal Server Error",
-            response = String.class
-        )
+            @ApiResponse(
+                    code = 200,
+                    message = "Representation of a User profile with their Roles",
+                    response = String.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid request has been provided",
+                    response = String.class
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "Not User Profile found with the given ID",
+                    response = String.class
+            ),
```

Comment:
403 status code missing

Reply:
403 for this endpoint and others have been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/302/commits/2b89ed9cd7068384338d596c65af7faf8429ca35